README.md

```diff
@@ -96,6 +96,10 @@ To install this tool run the following command:
 Now, we can run the package manager using the command `agda-pkg` or even
 shorter just `apkg`.
 
+## Using with Nix or NixOS
+
+A third-party [example repository](https://github.com/bbarker/LearningAgda) is available to create a `nix-shell` environment that loads `agda-pkg` as well as `agda` and `agda-mode` for emacs. 
```

Comment:
Could you please put the instructions here in the README, instead of having to navigate to another link?

Reply:
Sure, I inlined the docs and added to the ToC.

Reply:
Thanks! So, as far as I understand (I'm not nix user) if I put the files from your repository in my directory, from a nix-shell I can run the commands you provide?

Reply:
Yes, I can't think of any reason why that wouldn't work - sounds like a handy tool to have.